scm/driver/azure/pr.go

```diff
@@ -107,6 +107,7 @@ type pr struct {
 	SourceRefName         string      `json:"sourceRefName"`
 	TargetRefName         string      `json:"targetRefName"`
 	MergeStatus           string      `json:"mergeStatus"`
+	IsDraft               bool        `json:"isDraft"`
```

Comment:
can we call it draft at all places and just rename json names?
I see we follow that convention - if not we can keep isDraft.

Reply:
I read somewhere that the variable name need to start with uppercase letter if we expect it to be marshalled

Reply:
I meant something - but nvm - we are good